src/main/java/uk/nhs/digital/nhsconnect/nhais/service/EdifactToFhirService.java

```diff
@@ -1,14 +1,53 @@
 package uk.nhs.digital.nhsconnect.nhais.service;
 
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Interchange;
+
+import org.hl7.fhir.r4.model.Organization;
 import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.Patient;
+import org.hl7.fhir.r4.model.Practitioner;
+import org.hl7.fhir.r4.model.Reference;
+import org.hl7.fhir.r4.model.Resource;
 import org.springframework.stereotype.Component;
-import uk.nhs.digital.nhsconnect.nhais.model.edifact.Interchange;
 
 @Component
 public class EdifactToFhirService {
 
-    Parameters convertToFhir(Interchange interchange) {
-        return new Parameters();
+    public Parameters convertToFhir(Interchange interchange) {
+        Patient patient = createPatient("example");
```

Comment:
Do we know what should be the ID of patient resource? Left empty or a random GUID?

Reply:
Patient information will be mapped in another ticket - I just used a placeholder for now

Reply:
I think it's better to set something like "NOT_SET" value here than "example". Example looks like a remaining from not finished work

Reply:
Updated with value that "stands out" and TODO